modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache

```diff
@@ -128,40 +128,11 @@
       data
     end
 
-    # Save response body into a file in (the defined) temporary folder, using the filename
-    # from the "Content-Disposition" header if provided, otherwise a random filename.
-    # The response body is written to the file in chunks in order to handle files which
-    # size is larger than maximum Ruby String or even larger than the maximum memory a Ruby
-    # process can use.
-    #
-    # @see Configuration#temp_folder_path
     def download_file(request)
-      tempfile = nil
-      encoding = nil
-      request.headers do |response|
-        content_disposition = response.headers['Content-Disposition']
-        if content_disposition && content_disposition =~ /filename=/i
-          filename = content_disposition[/filename=['"]?([^'"\s]+)['"]?/, 1]
```

Comment:
@tsuwatch thanks for the PR. Would it be possible to keep the filename from the response headers if available?

Reply:
did it.
https://github.com/OpenAPITools/openapi-generator/pull/7842/commits/3e3ea88b0dec7763c861e9623f88a9d1fa9034c8